app/src/main/java/com/example/next_show/fragments/FeedFragment.java

```diff
@@ -61,59 +56,65 @@ public void onCreate(Bundle savedInstanceState) {
     @Override
     public View onCreateView(LayoutInflater inflater, ViewGroup container,
                              Bundle savedInstanceState) {
-        // Inflate the layout for this fragment
-        View currView = inflater.inflate(R.layout.fragment_feed, container, false);
+        // prevent reloading if not necessary
+        if(currView == null) {
+            // Inflate the layout for this fragment
+            currView = inflater.inflate(R.layout.fragment_feed, container, false);
 
-        // find Recycler View
-        rvFeed = currView.findViewById(R.id.rvFeed);
+            // find Recycler View
+            rvFeed = currView.findViewById(R.id.rvFeed);
 
-        // initialize the array that will hold posts and create a PostsAdapter
-        showsList = new ArrayList<>();
-        adapter = new ShowAdapter(getActivity(), showsList);
+            // initialize the array that will hold posts and create a ShowAdapter
+            showsList = new ArrayList<>();
+            adapter = new ShowAdapter(getActivity(), showsList, false);
 
-        // set the adapter on the recycler view
-        rvFeed.setAdapter(adapter);
+            // set the adapter on the recycler view
+            rvFeed.setAdapter(adapter);
 
-        // set the layout manager on the recycler view
-        rvFeed.setLayoutManager(new LinearLayoutManager(getActivity()));
+            // set the layout manager on the recycler view
+            rvFeed.setLayoutManager(new LinearLayoutManager(getActivity()));
 
-        // new application for Trakt Call, pass in the Context
-        Shows showsObj = new TraktApplication(getContext()).getNewShowsInstance();
+            // new application for Trakt Call, pass in the Context
+            Shows showsObj = new TraktApplication(getContext()).getNewShowsInstance();
 
-        // determine whether to show trending or recommended
-        BottomNavigationView bottomNavigationView = (BottomNavigationView) currView.findViewById(R.id.filterMenu);
+            // determine whether to show trending or recommended
+            BottomNavigationView bottomNavigationView = (BottomNavigationView) currView.findViewById(R.id.filterMenu);
 
-        // get trending shows
-        fetchTraktData(showsObj);
+            // get trending shows
+            fetchTraktData(showsObj);
```

Comment:
Can't you pull fetching the shows out of onCreateView and instead put it into `onCreate`?  Is that why you're trying to avoid totally setting up the views again?